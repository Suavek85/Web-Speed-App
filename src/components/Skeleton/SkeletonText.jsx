import React from 'react'
import Skeleton, { SkeletonTheme } from "react-loading-skeleton"

export default function SkeletonText() {
  return (
    <SkeletonTheme color="#bdbdbd" highlightColor="#d8d8d8">
      <Skeleton width={50} />
    </SkeletonTheme>
  )
}
 