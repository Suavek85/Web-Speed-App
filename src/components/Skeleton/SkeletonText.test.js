import React from 'react'
import { shallow } from 'enzyme'
import SkeletonText from './SkeletonText'

describe('SkeletonText snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SkeletonText />)
    expect(wrapper).toMatchSnapshot()
  })

})