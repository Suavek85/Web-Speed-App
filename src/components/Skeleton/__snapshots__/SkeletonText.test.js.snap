// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonText snapshot should match the snapshot 1`] = `
<SkeletonTheme
  color="#bdbdbd"
  highlightColor="#d8d8d8"
>
  <Skeleton
    circle={false}
    count={1}
    duration={1.2}
    height={null}
    width={50}
    wrapper={null}
  />
</SkeletonTheme>
`;
